app/assets/stylesheets/application.css.scss

```diff
@@ -93,9 +93,21 @@ article {
   color: rgba(14, 9, 8, 0.75);
   border: solid 3px #363636;
   border-radius: 5px 5px 5px 5px;
-  background-color: rgba(243, 243, 243, 0.87);
+  background-color: rgba(247, 247, 247, 0.87);
   padding: 5px;
-  margin: 25px 0 10px 0;
+  margin: 25px 80px 10px 50px;
+}
+
+#trip-title {
+  text-align: center;
```

Comment:
Properties should be ordered background-color, border, border-radius, color, font-family, font-weight, margin, padding, text-align